src/scene/webscene/index.jsx

```diff
@@ -58,10 +61,10 @@ class Webscene extends Component {
     if (prevProps.layerSettings !== this.props.layerSettings) {
       Object.keys(this.props.layerSettings).forEach(layerId => {
         const settings = this.props.layerSettings[layerId];
-        const layer = [
-          ...this.state.groupLayer.layers.items,
-          this.state.groundLayer,
-        ].find(l => l.id === layerId);
+        const layer =
+          this.state.groundLayer && this.state.groundLayer.id === layerId
+            ? this.state.groundLayer
+            : this.state.groupLayer.layers.items.find(l => l.id === layerId);
```

Comment:
Now the statement got even harder to read. What about:
```suggestion
        const layer =
          this.state.groundLayer && this.state.groundLayer.id === layerId
            ? this.state.groundLayer
            : this.state.groupLayer.layers.items.find(l => l && l.id === layerId);
```